mrp_product_use_case/wizard/mrp_product_use_case_wizard.py

```diff
@@ -0,0 +1,67 @@
+# -*- coding: utf-8 -*-
+
+from odoo import models, fields, api
+
+
+class MrpProductUseCaseWizard(models.TransientModel):
+    _name = 'mrp.product.use.case.wizard'
+
+    product_id = fields.Many2one('product.product', 'Component', required=True)
+
+    @api.multi
+    def mrp_product_use_case(self, wizard_id, component_id, level):
+        cr = self._cr
+        cr.execute("""
+            select
+                mbl.sequence,
+                mbl.product_id,
+                mbl.product_qty,
+                mb.product_tmpl_id,
+                mb.id mb_id,
+                (select id from product_product
+                 where product_tmpl_id=mb.product_tmpl_id limit 1) compose_id
+            from mrp_bom_line mbl inner join mrp_bom mb on mbl.bom_id=mb.id
+            where mbl.product_id=%s
+        """, (str(component_id),))
+        result = cr.fetchall()
+        for row in result:
+            compose_id = row[5]
+            level_txt = ''
+            for i in range(1, level):
+                level_txt = level_txt + u'-'
+            level_txt = level_txt + str(level)
+            vals = {
+                'wizard_id': wizard_id,
+                'level': level_txt,
+                'component_id': component_id,
+                'line': row[0],
+                'quantity': row[2],
+                'mrp_bom_id': row[4],
+            }
+            self.env['mrp.product.use.case.line'].create(vals)
+            self.mrp_product_use_case(wizard_id, compose_id, level + 1)
+
+    @api.multi
+    def do_search_component(self):
+        for obj in self:
+            if obj.product_id:
+                self.mrp_product_use_case(obj.id, obj.product_id.id, 1)
+        return {
+            'name': u"MRP Product Use Case " + str(obj.product_id.name),
```

Comment:
small improvement replace 
"MRP Product Use Case" by "Use Case" for a better display